app/src/main/res/layout-land/admin_pin_activity.xml

```diff
@@ -2,112 +2,97 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto"
   xmlns:profile="http://schemas.android.com/tools">
-
   <data>
-
     <variable
       name="viewModel"
       type="org.oppia.android.app.profile.AdminPinViewModel" />
   </data>
 
-  <FrameLayout
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/admin_auth_container"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:background="@color/oppiaBackgroundYellowIvory"
+    android:paddingBottom="108dp">
 
-    <ScrollView
```

Comment:
Please do not remove the `ScrollView` because considering that the app will be used on low-end devices and it might happen that the entire view does not fix in single screen, we should make this scrollable.

Also, that's one of the reasons why your tests are failing on CI.

<img width="747" alt="Screenshot 2020-10-10 at 10 54 54 AM" src="https://user-images.githubusercontent.com/9396084/95646538-0b67b080-0ae7-11eb-95e6-21df9ace8877.png">


One approach to solve this can be: 
`ScrollView` contains the `ConstraintLayout`

But if that does not work considering that `ScrollView` may create issues with `layout_height = match_parent`
then you can do following:

`FrameLayout` contains `ScrollView` contains the `ConstraintLayout`


Note: We have to flatten the view hierarchy as much as possible but not at the cost of based experience to users.

Thanks.

Reply:
My bad, I thought there was plenty of room and that the goal to flatten hierarchy was paramount.  Just researched the best practices right for embedded scrollviews with constraintlayout and will update this and my other PR to re-add scrollview.